python-checks/src/test/resources/checks/flaskHardCodedJWTSecretKey.py

```diff
@@ -38,30 +38,39 @@ def test_non_compliant_call_expressions():
     app = Flask(__name__)
 
     # Tests for "flask.app.Flask.config[.update]"
-    app.config.update(dict( # Noncompliant
```

Comment:
Given we are changing issues locations and messages, it makes sense to have assertions not only on primary locations, but also on secondaries and on messages.